quiz.js

```diff
@@ -0,0 +1,59 @@
+var heightInput = document.getElementById("height");
+var characterInput = document.getElementById("character");
+var submit = document.getElementById("submit");
+
+heightInput.addEventListener("input", treeFunction);
+characterInput.addEventListener("input", alertFunction);
+submit.addEventListener("click", submitFunction);
+
+
+function treeFunction() {
```

Comment:
This logic should be in the actualTreeFunction. If either height or character is empty, then stop the function. The way you have it written now, it will run actualTreeFunction even if the inputs are empty.